lib/auth/webauthncli/fido2.go

```diff
@@ -792,7 +817,7 @@ func handleDevice(
 		log.Debugf("FIDO2: Device %v filtered, err=%v", path, err)
 
 		// If the device is chosen then treat the error as interactive.
-		if waitErr := waitForTouch(dev); errors.Is(waitErr, libfido2.ErrNoCredentials) {
+		if touched, _ := waitForTouch(dev); touched {
```

Comment:
What happens if `dev.TouchBegin()` fails? 
We don't log or handle that error

Reply:
Well spotted, added logging.

Answering your question, if TouchBegin fails we won't be able to detect a touch, meaning that either the operation will be carried out by other plugged yubikeys or instantly abort (if all keys failed). I haven't yet seen this method fail in practice.